### DISCUSSION TITLE
This should be native tooo....... you guessed it ;)

### DISCUSSION THREAD

**Discussion by ghost** [Schneegans/Burn-My-Windows#167] [Ideas]
I just came here to say that after having switched to KDE Plasma; this extension is truly the only thing I miss about GNOME. This was hands down the best part about my desktop as I was able to really make it feel the way I wanted it too. 

I ended up trying KDE Plasma as I'd heard it was the environment to go to for ultimate customization; and I was not dissappointed. So much so in fact that I swapped to KDE Plasma on all 3 systems in the house! But woah is me that this is not available on KDE lol. I see many have inquired, and I"ve read the responses. 

I personally know very little about coding. I know just enough to maybe fix something should I find myself having broken something. But how I wish I knew how to collaberate with you to make this happen on KDE. 

Burn my Windows deserves to be native on the most customizable desktop environment I've ever seen; KDE lol! This extension is just that good. 

Anyway, whether BMW gets to KDE someday or not; I just wanted to thank you for your work; and contributing to great experiences while I was using GNOME.

**Comment by Schneegans** [comment]
Thank you for the kind words! And thanks to the [awesome work of Martin Flöser](https://invent.kde.org/plasma/kwin/-/merge_requests/2227) things have changed and _maybe_ you do not have to wait toooo long :wink:

**Reply by ghost** [reply]
You have NO IDEA how excited I am to hear this! Ya'lls creativity is something I thought only I thought would be cool. I've got all kinds of ideas but I lack the skills needed to do what you do lol. Keep it going, you guys are great!!!

**Reply by Schneegans** [reply]
A small teaser: https://www.reddit.com/r/kde/comments/va3xyw/a_small_teaser/ :wink:

**Comment by ghost** [comment]
This is some very exciting news!!! Being that I'm fairly new to KDE Plasma; is there a specific method that this will need be installed? Apologies for my ignorance!

A.O

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by Schneegans** [reply]
Well, it's not released yet, but you can install it from git, if you feel adventurous. There's a `feature/kde` branch with a kwin directory containing a build.sh script. You can execute this and copy the resulting files from the _build directory to .local/share/kwin/effects/. However, you'll need a fairly new version of kwin for them to work. I think at least 5.25.

**Comment by ghost** [comment]
No worries! I'm running Arch Linux and for the most part rocking the bleeding edge. I'll do some piddling around and see what I can figure out! Ya'll are awesome. Keep up the good work!

A.O

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
